src/lib/bm_ncp/ncp_config.cpp

```diff
@@ -165,3 +200,60 @@ static uint8_t* alloc_ncp_key_buffer(uint8_t num_keys, const ConfigKey_t* keys,
     return buf;
 }
 
+
+bool _cfg_get_bcmp_cb(uint8_t *payload) {
+    bool rval = true;
+    if (payload != NULL) {
+        bm_common_config_value_t *msg = reinterpret_cast<bm_common_config_value_t *>(payload);
+        if (bm_serial_cfg_value(msg->header.source_node_id, msg->partition, msg->data_length, msg->data) != BM_SERIAL_OK) {
+            printf("Failed to send get cfg response\n");
+            rval = false;
+        }
+    } else {
+        bm_printf(0, "Failed to get cfg\n");
```

Comment:
right now I am just printing out that we faile to get/set/status requests, but this ends up looking like this:
```
bm cfg get 910030066a2926a7 s sensorsPollIntervalMs
Succesfully sent config get msg
1709160098.746 4d58ce2f22dfb374, Failed to get cfg
```
which is not to pretty and doesn't make sense. So I am thinking that we should create a known "invalid" Node-ID that we send here with empty/garbage data. Then Spotter will know that if it recieves the "invalid" Node-ID, to print that the cfg get/set/status request timedout and failed. Thoughts? How about 0xffffffffffffffff as the "invalid" NODE-ID?

Reply:
Another option would be to store more data in the bcmp message requests linked list and assemble a response message, however, not all of the requests have a built in success/fail boolean.

That is why I thought of the "invalid" node-id since that is a common shared variable in each of the messages.

Reply:
Using `bm_printf` is fine for now, but in the future we should have a shared way of handling successful config requests and failed config requests.

Reply:
I'm going to switch it to use `BRIDGE_CFG_LOG_PRINT`